src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -168,7 +168,7 @@
              [components/amount-input
               {:error (when-not sufficient-funds? (i18n/label :t/wallet-insufficient-funds))
                :disabled? true
-               :input-options {:default-value amount}}]
+               :input-options {:default-value (str (money/wei->str :ether amount))}}]
```

Comment:
Why are we double `str`? Same above where we can use we->str fn, no?

Reply:
Good catch, shouldn't be `wei->str`